app/models/bookstore.rb

```diff
@@ -0,0 +1,14 @@
+class Bookstore < ActiveRecord::Base
+  STATES = [
```

Comment:
http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/MutableConstant

I guess you could add a `.freeze` to the end of the array, but I've never seen it.